src/etc/inc/dyndns.class

```diff
@@ -798,17 +798,28 @@
 					$dnsHost = str_replace(' ', '', $this->_dnsHost);
 
 					curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-					curl_setopt($ch, CURLOPT_HTTPHEADER, array(
-						'X-Auth-Email: '.$this->_dnsUser.'',
-						'X-Auth-Key: '.$this->_dnsPass.'',
-						'Content-Type: application/json'
-					));
 
-					// Get zone ID
-					$getZoneId = "https://{$dnsServer}/client/v4/zones/?name={$this->_dnsDomain}";
-					curl_setopt($ch, CURLOPT_URL, $getZoneId);
-					$output = json_decode(curl_exec($ch));
-					$zone = $output->result[0]->id;
+					if (strpos($this->_dnsUser, '@') !== false){
+						curl_setopt($ch, CURLOPT_HTTPHEADER, array(
+							'X-Auth-Email: '.$this->_dnsUser,
+							'X-Auth-Key: '.$this->_dnsPass,
```

Comment:
Should have a space on either side of the `.` operator on both lines here.

Reply:
I did not put there because it wasn't there originally, I fixed all the concatenations in this file based on your instruction.